components/NavigationSliceZone.vue

```diff
@@ -43,6 +43,7 @@
 
 <script>
 import { mapState } from 'vuex'
+import { groupBy } from 'lodash'
```

Comment:
**Question :** Serait-il envisageable de passer outre l'ajout de Lodash vu le peu de fonctions de la librairie nécessaires au projet ?

Reply:
Vu que c'est importé par méthode ce n'est pas trop lourd, et la méthode groupBy en JS natif est assez verbeuse..
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_groupby
Pour cette méthode je pense qu'utiliser Lodash n'est pas un souci. 

Ou alors créer une méthode groupBy à partir du code natif JS.

Reply:
Je propose de laisser, le site allant devenir de + en + complexe, ça sera utile de l'avoir sous la main.
Mais il faut qu'on garde l'habitude d'import que ce dont on a besoin et pas tout lodash :)